src/custom/api/ipfs/index.ts

```diff
@@ -1,8 +1,35 @@
-import pinataSDK, { PinataPinResponse } from '@pinata/sdk'
 import { PINATA_API_KEY, PINATA_SECRET_API_KEY } from 'constants/ipfs'
 
-const pinata = pinataSDK(PINATA_API_KEY, PINATA_SECRET_API_KEY)
+export async function pinJSONToIPFS(file: any): Promise<any> {
+  const pinataUrl = `https://api.pinata.cloud/pinning/pinJSONToIPFS`
```

Comment:
Maybe this could be moved to a constants file?

Reply:
at least the base URL